paper.md

```diff
@@ -82,8 +82,8 @@ it has been used in a number of papers (see below).
 This package computes optimal stability functions for Runge-Kutta methods.
 Here optimal means that the stable step size is maximized for a given ODE
 spectrum.  The corresponding optimization problem is intractable under a
-direct implementation.  The package uses the algorithm developed in
-[@2012_optimal_stability_polynomials], which transforms the problem into a
+direct implementation.  The package uses the algorithm developed by
```

Comment:
I don't think it's correct to say an algorithm was developed by a paper (which is how I read this).  I prefer "in".

Reply:
Usually, these parts as they appear in the paper as

> the algorithm developed by Ketcheson and Ahmadia (2012)

Here, I read "Ketcheson and Ahmadia" as the authors who developed the algorithm and "(2012)" as a hint to the paper. Writing

> the algorithm developed by Ketcheson and Ahmadia

would be okay (if one does not need to cite a paper), wouldn't it?

Reply:
To clarify: I've removed the braces around the citation command, switching from `\citep` in LaTeX to `\citet`. For me, it's totally fine to have 

> the algorithm developed in [42]

but

> the algorithm developed in Ketcheson

looks strange.

Reply:
But if you read this sentence differently, I will change it, of course.

Reply:
As it is currently (in master) it says

> the algorithm developed in (Ketcheson & Ahmadia, 2012)

which I am happy with.  Because of the parentheses, the ampersand, and the year it is clear that what is referred to is a paper and not a person or group of people.

It would be fine to write

> the algorithm developed by Ketcheson & Ahmadia [15]

in which case we first refer to the authors and then cite the paper.  But I wouldn't do this in my own paper as I consider it bad manners to refer to yourself by name in your published papers (or slides; when a reference is necessary, mathematicians generally replace their own name with just initials there).

The phrase

> the algorithm developed by Ketcheson and Ahmadia (2012)

is confusing to me because it's not clear what words belong to the text and what belongs to the citation.  I can't parse it in a way that makes sense.

Reply:
Okay. I've updated that in the branch for this PR in my fork (https://github.com/ranocha/RK-opt/commit/3ce6f99f0989d8b9001c348324bc571aa5bbdefa). However, that commit doesn't show up here...

Reply:
I've opened a new PR (#48) including the new commit.